build-docs.js

```diff
@@ -0,0 +1,82 @@
+const fs = require('fs');
+const path = require('path');
+const mkdirp = require('mkdirp');
+const MarkdownIt = require('markdown-it');
+const hljs = require('highlight.js');
+
+const sourceRoot = 'docs';
+const outputRoot = 'public/docs';
+const entryPoints = [
+  'emulator-suite/_includes/_firestore_instrument',
+  'emulator-suite/_includes/_firestore_admin_quickstart',
+  'emulator-suite/_includes/_rtdb_instrument',
+  'emulator-suite/_includes/_rtdb_admin_quickstart',
+  'emulator-suite/_includes/_functions_instrument',
+];
+
+const md = new MarkdownIt({
+  html: true,
+  highlight: function(str, lang) {
+    if (lang && hljs.getLanguage(lang)) {
+      try {
+        return hljs.highlight(lang, str).value;
+      } catch (err) {
+        console.warn(err);
+      }
+    }
+
+    return ''; // use external default escaping
+  },
+});
+
+entryPoints.forEach(entryPoint => {
+  const inputFile = path.join(__dirname, sourceRoot, entryPoint + '.md');
+  const outputFile = path.join(__dirname, outputRoot, entryPoint + '.html');
+
+  const source = fs.readFileSync(inputFile, 'utf-8');
+  let result = md.render(preProcess(source, inputFile));
+  result =
+    `<link
+  rel="stylesheet"
+  href="https://www.gstatic.com/devrel-devsite/prod/v2e3f09d6e6536badfdb5bf4153d08404c10f0bdcdc9056b4896a90327dc2c4ff/firebase/css/app.css"
```

Comment:
This is obviously not sustainable -- for POC styling only. Will find another way later.